payment/tests/test_payment.py

```diff
@@ -212,7 +229,7 @@ def test_utils_get_course(self):
     @patch('payment.utils.get_order')
     def test_confirmation_email_language(self, get_order_mock):
         get_order_mock.return_value = self.api_response
-        send_confirmation_email(self.user, 'FUN-100056')
+        send_confirmation_email(self.user, ecommerce_order_api_response(self.course.key))
```

Comment:
is it not `self.api_response` since this is what you return for `get_order` returns?
```suggestion
        send_confirmation_email(self.user, self.api_response)
```

Reply:
Indeed.